guides/release/tutorial/01-orientation.md

```diff
@@ -93,10 +104,9 @@ super-rentals
 ├── .template-lintrc.js
```

Comment:
https://github.com/ember-learn/super-rentals-tutorial/pull/65 did not work as intended. I am expecting a package-lock.json here as well.